docs/src/components/Navigation.js

```diff
@@ -2,10 +2,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Text, Box, SelectList, Link, Icon, Heading } from 'gestalt';
+import routes from '../routes';
 
-type Props = {|
-  cards: *,
-|};
+type Props = {||};
```

Comment:
@chrislloyd What does this mean? That no props should be passed to it?

Reply:
yeah, could probably use void here. Will revisit